lib/tbd/ua.rb

```diff
@@ -235,7 +235,7 @@ def uprate(model = nil, s = {}, argh = {})
       end
 
       if coll.empty?
-        log(ERR, "No construction to uprate - skipping (#{mth})")
+        log(ERR, "No #{label} construction to uprate - skipping (#{mth})")
```

Comment:
The log message is now a bit more informative on the nature of the ERROR (e.g. when a user explicitly asks to [uprate](https://rd2.github.io/tbd/pages/ut.html) exposed floors, when an OpenStudio model doesn't even hold any such exposed floor).